build/utils.ts

```diff
@@ -0,0 +1,49 @@
+import path from 'path'
+import os from 'os'
+import { getPackagesSync } from '@lerna/project'
+
+import { compRoot } from './paths'
+import chalk from 'chalk'
```

Comment:
I think this like should be above line#3

Reply:
Done.